scarlet/psf.py

```diff
@@ -253,8 +255,8 @@ def get_loss(self, model):
         relative error.
         """
         model_ = self.render(model)
-        images_ = self.images[self.slices]
-        weights_ = self.weights[self.slices]
+        images_ = self.images[self.slices_for_images]
+        weights_ = self.weights[self.slices_for_images]
```

Comment:
I'm not sure that I understand the need for this change, having both a `slices` and `slices_for_images` property. I agree that `slices_for_images` is more clear, but in that case why keep `slices` at all?